Scripts/Modules/SnappingModule/Shaders/GroundPlane.shader

```diff
@@ -0,0 +1,217 @@
+﻿Shader "EditorVR/GroundPlane"
```

Comment:
This is used on the transparent plane that shows up when an object is currently ground snapping. A normal Standard shader or existing custom shader either draws on top of everything or has a lot of z fighting and flicker when the scene has flat ground objects.